client/src/bin/ipc.rs

```diff
@@ -0,0 +1,25 @@
+use client::{register_cli_command, register_server_routes, HealthCheckHandler};
+use lazy_static::lazy_static;
+
+// initialize your server RPC handlers here
+lazy_static! {
+    static ref HEALTH: Arc<HealthCheckHandler> = Arc::new(HealthCheckHandler {});
```

Comment:
Why does this need to be static?

Reply:
just make it easier to initialize for now, I will move them to some initialization macro, maybe that's better